monitoring/configmap-reference-for-cluster-monitoring-operator.adoc

```diff
@@ -587,3 +587,5 @@ component.
 |thanosRuler |link:#thanosrulerconfig[ThanosRulerConfig] |ThanosRuler
 defines configuration for the Thanos Ruler component
 |===
+
+// TODO: What to do about these 1.24 instances. This file is only in main and 4.10, can we delete it from main?
```

Comment:
From this PR https://github.com/openshift/openshift-docs/pull/48344. Need to investigate

Reply:
Likely will be removed from main, so no changes needed w/ this PR